inst/using-the-dccvalidator-app-amp-ad.Rmd

```diff
@@ -46,25 +52,26 @@ requirements, validating and reviewing the metadata, and uploading the dataset.
 
 ## General Process Overview
 
-1. Contact the AMP-AD team to discuss the study and the expected data. Receive
+1. Contact the Data Liaison (juliane.schneider@sagebase.org) to discuss the study and the expected data. Receive
 staging folder synIDs for each expected dataset.
-2. Upload documentation and validate metadata + manifest files in [dccvalidator](https://shinypro.synapse.org/users/kwoo/dccvalidator-app/).
-3. Contact the AMP-AD team when all files pass validation. The team will verify
+2. Using the templates located at the bottom of the [Onboarding page](https://www.synapse.org/#!Synapse:syn24986618), create a study description, methods description(s), and acknowledgement statement.
+3. Upload the study description, methods description and acknowledgement statement using the [Submission Form](https://www.synapse.org/#!Synapse:syn25051271)
+4. Validate metadata + manifest files in [dccvalidator](https://shinypro.synapse.org/users/kwoo/dccvalidator-app/).
+5. Contact the AMP-AD team when all files pass validation. The team will verify
 items not checked by the dccvalidator. Receive permissions to upload data to the
 staging folder.
-4. Use the validated manifest to upload the data with `syncToSynapse` (see
+6. Use the validated manifest to upload the data with `syncToSynapse` (see
 [Synapse documentation](https://docs.synapse.org/articles/uploading_in_bulk.html) for uploading data in bulk).
```

Comment:
```suggestion
[Synapse documentation](https://docs.synapse.org/articles/uploading_in_bulk.html#validate-the-manifest-and-upload-files) for uploading data in bulk).
```

Reply:
This should probably put them right at the spot they need instead of the top of the page.

I am also wondering if it should be mentioned that the Python client will upload significantly faster than the R client here.

Reply:
@avanlinden Good idea! I'll add that.